autopush-common/src/lib.rs

```diff
@@ -46,10 +48,10 @@ pub mod util;
 /// "abandoned" and any router info assigned to a User Agent that has not contacted
 /// Autopush in 60 days can be discarded.
 ///
-const ONE_DAY_IN_SECONDS: u64 = 24 * 60 * 60;
+pub const ONE_DAY_IN_SECONDS: u64 = Duration::days(1).num_seconds() as u64;
```

Comment:
Huh. I thought the idea was that we'd make these `Duration` values, and only convert to seconds when we need to.

e.g. 
`.map(|ttl| min(ttl, MAX_NOTIFICATION_TTL.num_seconds() as i64)`

Basically, strongly type TTL as a `Duration` so that we don't muck up how we work with them accidentally.

Reply:
I believe Taddes is working on that (so this is currently a WIP)

Reply:
Ah, sorry! Hope you don't mind, but I converted this to a draft for now so I don't do more dumb drive-by's. 

My apologies.

Reply:
Yeah, sry, should have moved back to WIP. Wrestling with Prometheus all day and didn't put it back.